docs/it/userinfo_endpoint.rst

```diff
@@ -22,9 +22,9 @@ Lo UserInfo Endpoint deve supportare l’uso del solo metodo HTTP GET :rfc:`2616
 Response
 ++++++++
 
-La response dello UserInfo Endpoint deve specificare nel “Content-Type” il valore “application/jwt”.
+La response dello UserInfo Endpoint deve specificare nel "Content-Type" il valore "application/jwt".
 
-Il contenuto trasmesso nel body della Response deve essere un JWT firmato e cifrato secondo le modalità definite dall’Agenzia per l’Italia Digitale.
+Il contenuto trasmesso nel body della Response deve essere un JWT firmato e cifrato secondo le modalità definite dall'Agenzia per l'Italia Digitale.
```

Comment:
"... secondo le modalità definite dall'Agenzia per l'Italia Digitale." 
Di quale modalità stiamo parlando? dove è possibile reperirle?
@peppelinux  sarebbe possibile dare una lettura più chiara e immediata ?

Reply:
@sp640919 @damikael 

è possibile rifrasare con "deve essere Nested JWT firmato e cifrato come indicato in OIDC CORE" 
Perché di fatto SPID lo riporta as-is senza alcuna customizzazione

Reply:
non vedo il commit e la modifica al testo

Reply:
Ho riportato questo: 
Il contenuto del corpo della Response deve essere un JWT firmato e cifrato secondo le `modalità definite dall'Agenzia per l'Italia Digitale. <https://openid.net/specs/openid-connect-core-1_0.html#UserInfoResponse>`_